src/modules/apiOperations.js

```diff
@@ -0,0 +1,28 @@
+import { SHOW_BASE_URL, INVOLVEMENT_API_BASE_URL } from './default.js';
+
+export const getShows = async () => {
```

Comment:
I think the getshows method as from the requirements should have specified bounds, so please ensure when the page loads it displays a limited number of shows eg 6 show or 12 shows on page load instead of 240 shows